### PR TITLE
Fix typo in message name

### PR DESCRIPTION
**Pull request by benjamreis**
Signed-off-by: BenjiReis <benjamin.reis@vates.fr>

### CODE REVIEW

File: ocaml/xapi-consts/api_messages.ml [hunk 1]
```diff
@@ -100,7 +100,7 @@ let vm_resumed = addMessage "VM_RESUMED" 5L
 
 let vm_paused = addMessage "VM_PAUSED" 5L
 
-let vm_unpaused = addMessage "VM_UNPAUSE" 5L
+let vm_unpaused = addMessage "VM_UNPAUSED" 5L
```
[the remark this answers is not in the document]
Comment (danilo-delbusso) [reply ocaml/xapi-consts/api_messages.ml:103]: Referred to @kc284 to make sure this is answered correctly.

[the remark this answers is not in the document]
Comment (kc284) [reply ocaml/xapi-consts/api_messages.ml:103]: XenCenter relies on this value to detect the alert and localise it correctly. We can add the new value, but we should keep the old one too so we can still localise the alert for older servers.

[the remark this answers is not in the document]
Comment (benjamreis) [reply ocaml/xapi-consts/api_messages.ml:103]: I do'nt think this has yet be released in a Citrix Hypervisor. This value was introduced after 8.2 and never backported from what I can tell.

[the remark this answers is not in the document]
Comment (kc284) [reply ocaml/xapi-consts/api_messages.ml:103]: Hmm, actually, hold on, I need to check whether the new value can be added manually as these messages are autogenerated in the SDK.

[the remark this answers is not in the document]
Comment (kc284) [reply ocaml/xapi-consts/api_messages.ml:103]: Ah ok, if it's completely new we can change it. Other wise you would need to add these lines
```
            VM_UNPAUSE,
```
and
```
                    case "VM_UNPAUSE":
                        return MessageType.VM_UNPAUSE;
```
before lines 38 and 50 respectively in this file: https://github.com/xapi-project/xen-api/blob/master/ocaml/sdk-gen/csharp/templates/Message2.mustache

**Review by psafont — APPROVED** [review]

File: ocaml/xapi-consts/api_messages.ml (hunk 1 above)
[the remark this answers is not in the document]
Comment (benjamreis) [reply ocaml/xapi-consts/api_messages.ml:103]: I'm pretty sure its new. It came after 8.2 and has not been backported (perhaps someone at Citrix can confirm that) so I don't see how a CH host can already have this code.

**Review by lindig — APPROVED** [review]

**Pull request merged**